optimade/models/structures.py

```diff
@@ -389,6 +393,7 @@ class StructureResourceAttributes(EntryResourceAttributes):
     - A filter that matches an exactly given formula is `chemical_formula_hill="H2O2"`.""",
         support=SupportLevel.OPTIONAL,
         queryable=SupportLevel.OPTIONAL,
+        pattern=CHEMICAL_FORMULA_REGEXP,
```

Comment:
@CasperWA do you know where this even gets applied during validation? I seem to be able to get everything to pass...

Reply:
Nvm... should be using `regexp` not `pattern` :facepalm: